src/acquisition/covidcast/fill_is_latest_issue.py

```diff
@@ -0,0 +1,86 @@
+"""Computes and updates the `is_latest_issue` column in the `covidcast` table.
+
+This update is only needed to be run once.
+"""
+
+# third party
+import mysql.connector
+
+# first party
+import delphi.operations.secrets as secrets
+
+
+# partition configuration
+PARTITION_VARIABLE = 'geo_value'
+PARTITION_SPLITS = ["'05101'", "'101'", "'13071'", "'15007'", "'17161'", "'19039'", "'20123'", "'21213'", "'24035'",
```

Comment:
how did you come up with these split values?

Reply:
They were chosen so that they split a recent full issue into roughly equal partitions.